lib/registries/addon/discover/template.hbs

```diff
@@ -12,18 +12,7 @@
             value=(mut this.query)
```

Comment:
Change to angle bracket invocation since there is no body.

Reply:
@adlius, I made it so that it's not a block, but the conversion to angle bracket was fighting me for some reason and looks like it won't be super-quick. Is it worth updating that right now, or is fixing the actual incorrect thing good enough until we update the whole page?